EIPS/eip-4750.md

```diff
@@ -103,6 +99,8 @@ In addition to container format validation rules above, we extend code section v
 3. `RJUMP`, `RJUMPI` and `RJUMPV` immediate argument value (jump destination relative offset) validation:
     1. Code section is invalid in case offset points to a position outside of section bounds.
     2. Code section is invalid in case offset points to one of two bytes directly following `CALLF` instruction.
+4. First code section is invalid if it contains a `RETF` instruction.
+5. No unreachable sections are allowed, i.e. every section is referenced by at least one non-recursive `CALLF`, and section 0 is implicitly reachable.
```

Comment:
not sure if this belongs here or in 5450

Reply:
I would say here, as it's not related to stack validation.

Reply:
okay, good. I asked b/c I thought I've seen such checks included in the spec/algo for stack validation, maybe  I confused them with unreachable code within sections. Leaving this point in here then.